README.md

```diff
@@ -0,0 +1,83 @@
+# PyVista Tutorial
+
+## Abstract
+
+This tutorial will introduce 3D data visualization using PyVista (https://github.com/pyvista/pyvista).
+PyVista offers 3D plotting and mesh analysis through a streamlined interface for the Visualization Toolkit (VTK).
+PyVista uses VTK (http://www.vtk.org) under the covers and provides a very Python-friendly API.
+
+At the end of the tutorial users will be able to
+
+- Use PyVista effectively to create 3D visualizations. 
+- Understand the basic class structure of PyVista.
+
+## Keywords
+
+[visualization](https://github.com/topics/visualization)
+[meshviewer](https://github.com/topics/meshviewer)
+[vtk](https://github.com/topics/vtk)
+[3d](https://github.com/topics/3d)
+
+## Other Information and Files
+
+### Tutorial Description.
+Same as Abstract.
+
+### Tutorial Outline.
+
+0. Getting Started - Why PyVista? (10 min for talk, 5 min for train)
+
+1. Basic usage - Reading and plotting 3D data.  (10 min for talk, 5 min for train)
+
+2. What is a Mesh? - Learn about "Point", "Cell" and "Attributes". (10 min for talk, 5 min for train)
+
+3. PyVista Data Model - Learn about the each data model of the PyVista framework. (10 min for talk, 5 min for train)
+
+4. Mesh Creation - Train to build and plot meshes in PyVista. (10 min for talk, 5 min for train)
+
+5. Camera/Lights API - Train to use camera/light objects to customize your plot. (10 min for talk, 5 min for train)
+
+6. Charts API - Train to use the Charts API to plot your data in PyVista. (10 min for talk, 5 min for train)
+
+7. Unique uses of PyVista -  About some unique applications of PyVista. (10 min for talk, 5 min for train)
+
+8. External Examples - How PyVista is used in external project. (15 min for talk)
+
+### Additional Tutorial Information.
+
+We will base the material for the tutorial on [the examples in PyVista's online documentation](https://docs.pyvista.org/examples/index.html).
+The tutorial itself will be in the [pyvista-tutorial](https://github.com/pyvista/pyvista-tutorial) repository..
+
+### Tutorial Prerequisites.
+
+1. Basic knowledge of NumPy.
+
+2. Basic knowledge of SciPy.
+
+3. Basic knowledge of Matplotlib.
+
+4. It is ideal if you are a heavy user of VTK.
+
+## Appropriate level of the attendees' Python knowledge.
+
+- [ ] Beginner
+- [x] Intermediate
+- [ ] Advanced
+
+## Instructor Bio(s).
+
+### Tetsuo Koyama
+
+CAE software engineer in Japan.
```

Comment:
This is probably just my ignorance showing, but I don't know what CAE stands for, so it might be worth spelling out :)

Reply:
```suggestion
```